### DISCUSSION TITLE
additive accuracy option

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#13147] [Ideas] [answered]
additive accuracy works by adding an amount of accuracy for every judgement you get
additive accuracy will assume you will not hit anymore notes and will only increase if you any non-miss judgement

additive accuracy can be used to measure progress in the beatmap

**Comment by peppy** [comment]
Seems like a pointless duplicate of score display?

**Comment by iSimpFor** [comment]
in all modes, score works in a multiplicative way, this idea works accuracy into a additive display where combo doesn't influence said display
said display can be used as an raw score display or something like that

**Comment by bdach** [comment]
So this would be like the opposite of #9336?

**Comment by iSimpFor** [comment]
yes

**Comment by mk56-spn** [comment, marked as answer]
was closed by: https://github.com/ppy/osu/pull/21945
